### PR TITLE
added missing items to item_basic and item_equipment

### PR DESCRIPTION
**Pull request by JulianoW**
Item_basic and armors in item_equipment should be up to date, in terms of info available on dat's. Fixed some flags, and populated model id's where I could.

### CODE REVIEW

File: sql/item_basic.sql [hunk 1]
```diff
@@ -19610,6 +19615,7 @@ INSERT INTO `item_basic` VALUES (26484,0,'chefs_ecu_-1','chefs_ecu_-1',1,63552,0
 INSERT INTO `item_basic` VALUES (26485,0,'chefs_scutum_-1','chefs_scutum_-1',1,63552,0,0,0);
 INSERT INTO `item_basic` VALUES (26486,0,'chefs_shield_-1','chefs_shield_-1',1,63552,0,0,0);
 INSERT INTO `item_basic` VALUES (26517,0,'shadow_lord_shirt','shadow_lord_shirt',1,64584,0,0,0);
+INSERT INTO `item_basic` VALUES (26517,0,'jody_shirt','jody_shirt',1,63552,0,0,0);
```
Comment (TeoTwawki) [inline sql/item_basic.sql]: bad ID

Comment (JulianoW) [reply sql/item_basic.sql]: oops, my bad, >.< will keep a closer eye

**Comment by JulianoW** [comment]
My bad - fixed the item id. Also noticed the new items in item_basic are missing from item_equipment because I didn't import the fixes when running my sql compare. Gonna fix those up, make sure they commit, then recommit.

File: sql/item_equipment.sql [hunk 2]
```diff
@@ -12568,8 +12770,8 @@ INSERT INTO `item_equipment` VALUES (27382,'carmine_greaves',99,119,2204880,62,0
 INSERT INTO `item_equipment` VALUES (27383,'carmine_greaves_+1',99,119,2204880,62,0,0,256,0);
 INSERT INTO `item_equipment` VALUES (27384,'bewitched_leggings',99,0,2368481,100,0,0,256,0);
 INSERT INTO `item_equipment` VALUES (27385,'voodoo_leggings',99,0,2368481,100,0,0,256,0);
-INSERT INTO `item_equipment` VALUES (27386,'bewitched_schuhs',99,0,65,95,0,0,256,0);
-INSERT INTO `item_equipment` VALUES (27387,'voodoo_schuhs',99,0,65,95,0,0,256,0);
+INSERT INTO `item_equipment` VALUES (27386,'bewitched_schuhs',99,0,65,95,2,0,256,0);
```
Comment (TeoTwawki) [inline sql/item_equipment.sql]: are are we setting shieldSize 2 on feet?

Comment (TeoTwawki) [reply sql/item_equipment.sql]: *why are

Comment (JulianoW) [reply sql/item_equipment.sql]: sigh, I need a better way of putting these SQL statements together. A huge chunk I did programmatically which helped avoid mistakes, but damn if the stuff I adjusted manually wasn't littered with them. Oh well, live and learn.  (the 2 was supposed to be model id)

**Pull request merged**